src/testrender/cuda/optix_raytracer.cu

```diff
@@ -4,15 +4,27 @@
 
 
 #include <optix.h>
-
-#include "util.h"
-
 #include <optix_device.h>
 
 #include <OSL/hashes.h>
 
-#include "../render_params.h"
+#include "optix_raytracer.h"
 #include "rend_lib.h"
+#include "vec_math.h"
+
+#include "../background.h"
+#include "../bvh.h"
+#include "../raytracer.h"
+#include "../render_params.h"
+#include "../sampling.h"
+
+// clang-format off
+// These files must be included in this specific order
```

Comment:
Can we make `shading.cpp` include `shading.h` and only include the `.cpp` here?

Reply:
Yeah, this is another leftover from an earlier iteration.